src/Symfony/Component/Serializer/Serializer.php

```diff
@@ -173,7 +177,7 @@ public function normalize($data, $format = null, array $context = [])
             throw new NotNormalizableValueException(sprintf('Could not normalize object of type "%s", no supporting normalizer found.', \get_class($data)));
         }
 
-        throw new NotNormalizableValueException(sprintf('An unexpected value could not be normalized: %s.', !\is_resource($data) ? var_export($data, true) : sprintf('%s resource', get_resource_type($data))));
+        throw new NotNormalizableValueException(sprintf('An unexpected value could not be normalized: "%s".', !\is_resource($data) ? var_export($data, true) : sprintf('"%s" resource', get_resource_type($data))));
```

Comment:
fabbot fixed this lmk if I should revert

Reply:
looks good, the issue does not exist on lower branches

Reply:
I removed it from my PR as it's not related and was conflicting with the 4.4 branch